core/sqf/sql/scripts/install_local_hadoop

```diff
@@ -391,7 +391,19 @@ else
     MY_START_PORT=`expr 24000 + $MY_START_PORT '*' 200`
   elif [ $MY_START_PORT == "rand" ]; then
     # pick a random number between 12000 and 24000 that is divisible by 200
-    MY_START_PORT=`expr $RANDOM '%' 60 '*' 200 + 12000`
+    lv_number=0   #initialize
```

Comment:
Sorry, I don't quite understand the reason for this change. Isn't the new code doing what the old code did, just more complicated? Also, it extends the range up to 31200. We wanted to keep the range fro 24000 to 32000 reserved for installations that use the -p fromDisplay option. Please don't change this algorithm unless there is a very good reason for doing so.